ecgtools/parsers/cmip.py

```diff
@@ -74,7 +74,10 @@ def parse_cmip6(file):
             attributes['init_year'] = init_year
             attributes['start_time'] = start_time
             attributes['end_time'] = end_time
-            attributes['time_range'] = f'{start_time}-{end_time}'
+            if (start_time is None) & (end_time is None):
```

Comment:
We could simplify this condition to:

```suggestion
            if not (start_time and end_time):
```